src/main/resources/templates/index.html

```diff
@@ -28,7 +28,7 @@ <h1 class="text-center">
       <div class="form-group">
         <label for="abbreviation">Please add a name for your short link</label>
         <input class="form-control" id="abbreviation" name="abbreviation" placeholder="Abbreviation, e.g. 'foo'"
-               required th:errorclass="is-invalid" th:value="*{abbreviation}" type="text">
```

Comment:
Why are you making this field optional? Even when it is automatically populated, this should be required.